Sources/ApodiniREST/RESTExporterConfiguration.swift

```diff
@@ -19,6 +19,8 @@ extension REST {
         public let encoder: AnyEncoder
         /// The to be used `AnyDecoder` for decoding requests to the `RESTInterfaceExporter`
         public let decoder: AnyDecoder
+        /// How `Date` objects passed as query or path parameters should be decoded.
```

Comment:
Really picky: Remove the dot at the end, just for consistency

Reply:
I would argue to put dots everywhere to properly mark End of Sentence 🙃